docs/snippets/preact/button-story.js.mdx

```diff
@@ -0,0 +1,14 @@
+```js
+// Button.stories.js|jsx
+
+/** @jsx h */
+import { h } from 'preact';
+import { Button } from './Button';
+
+export default {
+    title: 'Button',
+    component: Button,
+};
```

Comment:
@fbredius if you could update the snippet and only use 2 spaces instead of 4 we'd appreciate it:
With that in mind the default export would be as such:
```js
export default {
  title: 'Button',
  component: Button,
}
```

This is basically to remain consistent, as some environments/setups might be configured for 4 spaces

Reply:
Thanks @jonniebigodes for your help! Have a great day as well.